testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -56,32 +56,32 @@
   text = Regexp.new(text) if type == 'regex'
   next if has_content?(text, wait: 3)
 
-  repeat_until_timeout(message: "Couldn't find text '#{text}'") do
-    break if has_content?(text, wait: 3)
+  begin
+    repeat_until_timeout(message: "Couldn't find text '#{text}'") do
+      break if has_content?(text, wait: 3)
 
-    begin
-      accept_prompt do
-        execute_script 'window.location.reload()'
-      end
-    rescue Capybara::ModalNotFound
-      # ignored
+      refresh_page
     end
+  rescue StandardError => e
```

Comment:
Probably a stupid question, but why not `rescue TimeoutError => e` ?

Reply:
Not stupid at all, I was actually asking myself the same question looking at this again after 1 week lol

`repeat_until_timeout` can raise 2 kind of errors and I wanted to intercept both of them, probably.

1) `raise ScriptError, "Giving up after #{attempts} attempts#{detail}" if attempts == retries`
2) `raise TimeoutError, "Timeout after #{timeout} seconds (repeat_until_timeout)#{detail}"`

But I wonder if it makes sense to have a ScriptError and TimeoutError; maybe they can be both labeled as TimeoutError